werkzeug/contrib/lint.py

```diff
@@ -126,15 +131,15 @@ class GuardedIterator(object):
 
     def __init__(self, iterator, headers_set, chunks):
         self._iterator = iterator
-        self._next = iter(iterator).next
+        self._next = iter(iterator).__next__
```

Comment:
We need to use something like `functools.partial(next, iter(iterator))` here. This has to run under Python 2 as well.